<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Button0">
      <property name="text">
       <string>getOpenFileName</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button1">
      <property name="text">
       <string>getOpenFileNames</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button2">
      <property name="text">
       <string>getSaveFileName</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button3">
      <property name="text">
       <string>getExistingDirectory</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button4">
      <property name="text">
       <string>QColorDialog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button5">
      <property name="text">
       <string>QFontDialog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button6">
      <property name="text">
       <string>QInputDialog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button7">
      <property name="text">
       <string>QMessageBox</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
